lib/components/artist/artworks/index.js

```diff
@@ -29,7 +30,9 @@ class Artworks extends React.Component {
       return this.renderSection('Works',
                                 other_count,
                                 this.props.artist.not_for_sale_artworks,
-                                (artistID, page) => artworksQuery(artistID, 'IS_NOT_FOR_SALE', page))
+                                () => { return ({ availability: 'IS_NOT_FOR_SALE'}) },
+                                this.resolveQuery,
+                                 null)
```

Comment:
Now looks like: `renderSection(title: string, count: number, artworks: any[], stateQuery: any, pageQuery: any, onComplete: any)`

interesting points are `stateQuery: any, pageQuery: any` - now all of the metadata for a query can be held inside the host component for the infinite gridview